lib/internal/errors.js

```diff
@@ -172,6 +172,10 @@ E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
 E('ERR_STDERR_CLOSE', 'process.stderr cannot be closed');
 E('ERR_STDOUT_CLOSE', 'process.stdout cannot be closed');
 E('ERR_STREAM_WRAP', 'Stream has StringDecoder set or is in objectMode');
+E('ERR_TLS_CERT_ALTNAME_INVALID',
+  (reason) => {
+    return `Hostname/IP does not match certificate's altnames: ${reason}`;
+  });
```

Comment:
actually that is what I had originally.  The suggestion to use a function was from James to avoid having to escape the '.  Having said that I think I prefer avoiding the function for consistency so I'll change back.